config/resources.yaml

```diff
@@ -1,10 +1,6 @@
 default_resources:
-  threads: 1
-  time: "12:00:00"
   mem_mb: 1024
-  mem_per_cpu: 3072
+  mem_per_cpu: 1024
   partition: core
-
-fastqc:
-  threads: 10
-  time: "02:00:00"
+  threads: 1
+  time: "1:00:00"
```

Comment:
Remove:
threads: 1
time: "1:00:00"

Reply:
I checked again and I am very confused. In `default_resources`, we should have both `threads` and `time`